public/js/app/util/helpers.js

```diff
@@ -91,21 +91,21 @@ export const emptyCheckboxes = className => {
 }
 
 export const sortRoundAndKoppsInfo = (roundKopps, prevSortedInfo) => {
-  const { ladokRoundId } = roundKopps
-  const { sortedRoundIds, sortedKoppsInfo } = prevSortedInfo
-  sortedRoundIds.push(ladokRoundId)
-  const sortedRounds = sortedRoundIds.sort()
-  const addIndex = sortedRounds.indexOf(ladokRoundId)
+  const { applicationCodes } = roundKopps
+  const { sortedApplicationCodes, sortedKoppsInfo } = prevSortedInfo
+  sortedApplicationCodes.push(applicationCodes[0])
```

Comment:
safe extracting of applicationCodes[0], we dont want to push undefined

Reply:
Changed array to direct string. Now round.applicationCode will be used.